src/user/announcements.rst

```diff
@@ -9,6 +9,23 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 ----
 
 
+2022-08-11: Moving to Visual Studio toolchain vc142
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Microsoft has deprecated the Visual Studio (VS) 2017 compiler and removed it
+from all the CI they control (notably Azure Pipelines & Github Actions).
+This means that the default toolchain (== C++ compiler, linker, standard
+libraries, and related utilities) of that VS version - vc141 - is getting less
+and less use in upstream libraries (because their public CI doesn't use it
+anymore without a lot of fiddling), and therefore support for it is bitrotting
+at an accelerating pace. We are therefore planning to move our toolchain on
+windows to vc142 (the default in VS2019) in two weeks, on 2022-08-25.
+
+This will not affect you as a general user of conda-forge packages on windows;
+the only impact is that if you are locally compiling against artefacts produced
+by conda-forge and are still using VS2017 yourself, you will need to upgrade your
+compiler (VS2019 is a drop-in & ABI-compatible).
```

Comment:
Please add a link to the PR in conda-forge-pinning-feedstock

Reply:
Done.